packages/appstore/src/components/modals/open-positions-svg-modal/open-positions-svg-modal.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv/components';
+import { useMT5SVGEligibleToMigrate } from '@deriv/hooks';
+import {
+    CFD_PLATFORMS,
+    getCFDPlatformLabel,
+    getFormattedJurisdictionCode,
+    Jurisdiction,
+    JURISDICTION_MARKET_TYPES,
+} from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+
+type TOpenPositionsSVGModal = {
+    market_type: string;
+    open_order_position_status: boolean;
+    is_modal_open: boolean;
+    setModalOpen: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const OpenPositionsSVGModal = ({
+    market_type,
+    open_order_position_status,
+    is_modal_open,
+    setModalOpen,
+}: TOpenPositionsSVGModal) => {
+    const { eligible_account_to_migrate_label } = useMT5SVGEligibleToMigrate();
+    const title = open_order_position_status ? 'No new positions' : 'Account closed';
+    const account_type =
+        market_type === JURISDICTION_MARKET_TYPES.FINANCIAL
+            ? JURISDICTION_MARKET_TYPES.FINANCIAL
+            : JURISDICTION_MARKET_TYPES.DERIVED;
+    const from_account = getFormattedJurisdictionCode(Jurisdiction.SVG);
+    const cfd_platform = getCFDPlatformLabel(CFD_PLATFORMS.MT5);
+    const onClick = () => {
+        setModalOpen(false);
+    };
+    return (
+        <Modal
+            is_open={is_modal_open}
+            toggleModal={() => setModalOpen(is_modal_open => !is_modal_open)}
+            small
+            has_close_icon={false}
+        >
+            <Modal.Body>
+                <Text as='h1' color='prominent' weight='bold' className='open-positions-svg__modal-title'>
+                    <Localize i18n_default_text='{{title}}' values={{ title }} />
+                </Text>
+                <Text as='p' color='prominent ' size='xs' line_height='m'>
```

Comment:
```suggestion
                <Text as='p' color='prominent ' size='xs'>
```
We can remove this `line_height` since the default value is 'm'